hack/generate/samples/ansible/memcached_molecule.go

```diff
@@ -120,13 +120,8 @@ func ImplementMemcachedMolecule(sample sample.Sample, image string) {
 	pkg.CheckError("adding watch_namespace_patch.yaml", err)
 
 	log.Info("adding WATCH_NAMESPACE env patch to patch list to be applied")
-	err = kbutil.InsertCode(filepath.Join(sample.Dir(), "config", "testing", "kustomization.yaml"), "patchesStrategicMerge:",
-		fmt.Sprintf("\n- %s", watchNamespacePatchFileName))
+	err = kbutil.InsertCode(filepath.Join(sample.Dir(), "config", "testing", "kustomization.yaml"), "patches:",
+		fmt.Sprintf("\n- path: %s", watchNamespacePatchFileName))
 	pkg.CheckError("inserting in kustomization.yaml", err)
 
-	log.Info("enabling metrics in the manager")
-	err = kbutil.UncommentCode(
-		filepath.Join(sample.Dir(), "config", "default", "kustomization.yaml"),
-		"#- path: manager_metrics_patch.yaml", "#")
-	pkg.CheckError("enabling metrics endpoint", err)
```

Comment:
I think these lines got inadvertently removed which were added as part of https://github.com/operator-framework/ansible-operator-plugins/pull/90

Reply:
I'm not quite sure if this should have remained or removed. My reasoning for removing was because there was an error saying that the commented section was not found. I thought it was because it is already uncommented. Also, I wasn't able to find where this was being uncommented, hence I removed it. Should this be added back?

cc: @joelanford

Reply:
@arkadeepsen metrics is now enabled by default in `kubebuilder` so there is nothing that needs to uncomment. See main where the kustomize file has this enabled.

https://github.com/operator-framework/ansible-operator-plugins/blob/main/testdata/memcached-molecule-operator/config/default/kustomization.yaml#L30-L32

Reply:
Ack. Thanks for the clarification.